pkg/console/install_panels.go

```diff
@@ -2,7 +2,7 @@ package console
 
 import (
 	"fmt"
-	"net"
+	netutil "k8s.io/utils/net"
```

Comment:
3rd party import should be listed on the second import group

Reply:
Thanks. Fix it by force-push the PR>